src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -104,6 +104,9 @@ static bool mpuSpi6000InitDone = false;
 
 static IO_t mpuSpi6000CsPin = IO_NONE;
 
+#define SPI_CLOCK_MPU6000_INITIALIZATION SPI_CLOCK_INITIALIZATON
+#define SPI_CLOCK_MPU6000_FAST           SPI_CLOCK_FAST
```

Comment:
Also, if the APB1 are run at 36 MHz and APB2 at 72MHz - their maximum frequencies - the the divider value "2" means 18 Mhz on APB1 and 36 MHz on APB2. Your constants are not so universal as you indicate they are.
Or am I all wrong?